cernopendata_client/cli.py

```diff
@@ -72,7 +73,6 @@ def get_metadata(server, recid, doi, title, output_value):
     Select a CERN Open Data bibliographic record by a record ID, a
     DOI, or a title and return its metadata in the JSON format.
 
-    \b
```

Comment:
We need to keep `\b` otherwise the formatting of help is not right; please check `cernopendata-client download-files --help`.

You can silence docstring formatting warning by `# noqa: D301`, see `reana-client`.

Reply:
ohh okay, I will add it. 👍